### ISSUE TITLE
Loading dialogs are dismissable

### ISSUE THREAD

**Issue by wasdennnoch** [k0shk0sh/FastHub#560]
**App Version: 2.5.1**

**OS Version: 25**

**Model: samsung-GT-I9305**

 When clicking outside a loading dialog with the circular progressbar it dismisses it, leaving you with a blank page and the request loading in the background.

I'd suggest either making it undismissable but then the user couldn't go back while it's loading; a more elegant solution is to extract the loading bar into the started activity itself so instead of just a blank page with a blank header and dialog on top you see a progressbar with a blank header. I hope that's understandable.

**Comment by yakov116** [comment]
@k0shk0sh Is this a duplicate of #471?

**Comment by wasdennnoch** [comment]
Not entirely; this issue seems to more refer to the readme loading. It may be on purpose but I say it's a bad design choise :P

**Comment by JediBurrell** [comment]
@wasdennnoch the README page has a progressbar even if you dismiss the dialog.

**Comment by wasdennnoch** [comment]
Only after the repository info itself loaded. And issues don't have any other loading indicator.
